web/src/info_overlay.ts

```diff
@@ -359,3 +360,9 @@ export function show(target: string | undefined): void {
         toggler!.goto(target);
     }
 }
+
+function process_message_formatting_layout($rendered_html: JQuery): void {
```

Comment:
I wonder if this shouldn't be renamed and placed in `rendered_markdown.ts`. It's possible that this might be a useful function to have elsewhere, and it seems likelier to be overlooked in `info_overlay.ts`.